sale_order_general_discount/views/sale_order_view.xml

```diff
@@ -7,8 +7,8 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='amount_tax']" position="after">
-                <field name="general_discount" />
+            <xpath expr="//field[@name='tax_totals_json']" position="before">
+                <field name="general_discount" class="oe_subtotal_footer" />
```

Comment:
Why is this put on the footer instead on the header part?

Reply:
> Why is this put on the footer instead on the header part?

It's the original design and the migration doesn't change that.

Reply:
OK, it seems a bit weird that something changed on the footer changes the lines, but it seems this way since at least 13.0.